.github/DISCUSSION_TEMPLATE/questions.yml

```
@@ -0,0 +1,37 @@
+name: "[Questions] Questions and/or support"
+description: Ask for help from the developers
+labels: [ "support" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for opening this discussion!
+        Before you submit, please make sure you have read our [Code of Conduct](https://github.com/Ouranosinc/xclim/blob/master/CODE_OF_CONDUCT.md).
+  - type: textarea
+    id: setup-information
+    attributes:
+      label: Setup Information
+      description: |
+        What software versions are you running? Example:
+        * Xclim version: 0.55.0-gamma
+        * Python version: 4.2
+        * Operating System: Nutmeg Linux 12.34 | macOS 11.0 "Redmond"
+      value: |
+        * Xclim version:
+        * Python version:
+        * Operating System:
```

```suggestion
        What xclim version are you running? 
      value: |
        * Xclim version:
```

---

@huard would you be amenable to something more professional, but still asking for the same information? I find that knowing whether the user is running an older version of Python or on Windows is important to identifying edge cases.

```suggestion
        What is your system information?
      value: |
        * Xclim version:
        * Python version:
        * Operating System:
```